tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -5563,3 +5563,70 @@ def test_readable_device_name(self):
         assert len(data) == 1
         assert data[0]["device"] == "iPhone14,3"
         assert data[0]["readable"] == "iPhone 13 Pro Max"
+
+    def test_http_status_code(self):
+        project1 = self.create_project()
+        project2 = self.create_project()
+        self.store_event(
+            data={
+                "event_id": "a" * 32,
+                "transaction": "/example",
+                "message": "how to make fast",
+                "timestamp": self.ten_mins_ago_iso,
+                "user": {"email": "cathy@example.com"},
+                "tags": {"http.status_code": "200"},
+            },
+            project_id=project1.id,
+        )
+        self.store_event(
+            data={
+                "event_id": "b" * 32,
+                "transaction": "/example",
+                "message": "how to make fast",
+                "timestamp": self.ten_mins_ago_iso,
+                "user": {"username": "catherine"},
+                "contexts": {"response": {"status_code": 400}},
+            },
+            project_id=project2.id,
+        )
+
+        features = {"organizations:discover-basic": True, "organizations:global-views": True}
+        query = {
+            "field": ["event.type", "http.status_code"],
+            "query": "",
+            "statsPeriod": "24h",
+        }
+        response = self.do_request(query, features=features)
+        assert response.status_code == 200, response.content
+        data = response.data["data"]
+        assert len(data) == 2
+        result = {r["http.status_code"] for r in data}
+        assert result == {"200", "400"}
+
+    def test_http_status_code_context_priority(self):
+        project1 = self.create_project()
+        self.store_event(
+            data={
+                "event_id": "a" * 32,
+                "transaction": "/example",
+                "message": "how to make fast",
+                "timestamp": self.ten_mins_ago_iso,
+                "user": {"email": "cathy@example.com"},
+                "tags": {"http.status_code": "200"},
+                "contexts": {"response": {"status_code": 400}},
+            },
+            project_id=project1.id,
+        )
+
+        features = {"organizations:discover-basic": True, "organizations:global-views": True}
+        query = {
+            "field": ["event.type", "http.status_code"],
+            "query": "",
+            "statsPeriod": "24h",
+        }
+        response = self.do_request(query, features=features)
+        assert response.status_code == 200, response.content
+        data = response.data["data"]
+        assert len(data) == 1
+        result = {r["http.status_code"] for r in data}
```

Comment:
nit:

```suggestion
        assert data[0]["http.status_code"] == "400"
```

Since we already checked there's only 1 item on the previous line

Reply:
Yeah, missed this cuz of the quick copy paste mentality haha

Reply:
Fixed.